Editor/Window/RegisterComputeInput.cs

```diff
@@ -54,7 +54,9 @@ public RegisterComputeInput(VisualElement container, StateManager stateManager)
             RegisterCallbacks();
             SetupConfigSettings();
             SetupStatusBox();
-            
+
+            _stateManager.OnUserProfileUpdated += OnRegisterComputeButtonClicked;
```

Comment:
OnRegisterComputeButtonClicked takes actions to register a compute / re-register a compute etc based on a user click. This line essentially means all of that will be performed when a user profile is updated? That seems redundant to me. We should only refresh the UX as required in such a scenario. And as part of the refresh, whatever needs to be fetched should be fetched from the service.

Reply:
Moved from here to just doing it on fleet id updating.